src/embed.ts

```diff
@@ -351,17 +357,17 @@ async function _embed(
     }
   }
 
-  await view.initialize(el).runAsync();
+  await view.initialize(chartWrapper).runAsync();
```

Comment:
What happens if the action menu is disabled?

Reply:
the wrapper will fill the remaining space, so there shouldn't be an issue. Is that what you are asking?

Reply:
example: 
![image](https://user-images.githubusercontent.com/11954298/85906209-77393e00-b7c2-11ea-8802-309a5bfca91b.png)

Reply:
Can we not add the wrapper when there is no action button?

Reply:
Sure. I made the change.